src/AppInstallerCommonCore/Architecture.cpp

```diff
@@ -46,7 +83,6 @@ namespace AppInstaller::Utility
         switch (systemInfo.wProcessorArchitecture)
         {
         case PROCESSOR_ARCHITECTURE_AMD64:
-        case PROCESSOR_ARCHITECTURE_IA64:

```

Comment:
>PROCESSOR_ARCHITECTURE_IA64 [](start = 13, length = 27)

Is this removal intended?

Reply:
Yes, IA64 is Itanium, which is *not* AMD64.  If anything it would be its own value, but I don't think we need to support it unless there is large customer demand.

---
In reply to: [511303758](https://github.com/microsoft/winget-cli/pull/626#discussion_r511303758) [](ancestors = 511303758)